### ISSUE TITLE
How do I remove the triangles from the camera view

### ISSUE THREAD

**Issue by ranbuch** [paulmg/ThreeJS-Webpack-ES6-Boilerplate#3]
There are orange, black, green and blue triangles on the scene.

How can I remove them?

Thank you.

**Comment by paulmg** [comment]
Hey @ranbuch, I just booted up the project in chrome on windows and not seeing these triangles in the scene. Could you provide me with a screenshot and also the OS and browser that you are using? Then I can maybe see about resolving your issue. Thanks

**Comment by ranbuch** [comment]
Thank you for your answer @paulmg 
This is what I'm walking about:
![capture](https://user-images.githubusercontent.com/3777546/36634337-b5333aec-19ab-11e8-82be-f846a878cded.PNG)

**Comment by ranbuch** [comment]
Deleting the shadow section from the config did the trick.
Don't worry about the rest of the scene, I've removed it as well.
I do, however, have a problem with GLTF loader and I belie this problem will occur in many THREE-JS plugins as they are depending on the face that `THREE` is a global variable and in your case `THREE` is a private variable.

What I've done is copying the [loader](https://threejs.org/examples/js/loaders/GLTFLoader.js)'s content, witch start like that:
```
THREE.GLTFLoader = ( function () { 
. . . 
```
and wrap it on a class like that:
```
export default class GLTFLoader {
    constructor(THREE) {
        THREE.GLTFLoader = ( function () {
        . . .
```
Than, on the `main.js` constructor:
```
new GLTFLoader(THREE);
```

Although everything ran without any errors the GLTF 3D model wasn't visible on the scene (0 errors and the success callback was triggered).

I thought it was a zoom issue and maybe I'm just not looking towards the right direction but even afer allowing zoom out `minDistance` `maxDistance` `minPolarAngle` `maxPolarAngle` `minAzimuthAngle` etc . . . I couldn't find it.

I tried to use the loader outside your boilerplate and it has worked easily.

Any suggestions?

And thank you for your last answer!

**Comment by paulmg** [comment]
I did something like this for the DDS Loader when I had to import it into the project:

```
DDSLoader.prototype = Object.create(THREE.CompressedTextureLoader.prototype);
DDSLoader.prototype.constructor = DDSLoader;
```

Your method should work as well though, I believe. Not sure why you aren't seeing the model and honestly it could be a myriad of reasons why. One thing is to check the scale of the model in the config. Make sure it's set to a scale that works with what you are importing. Also console log and check the network tab to ensure that it is actually both getting imported and inserted into the scene. Hope that helps.
